blueprints/reserved-capacity/README.md

```diff
@@ -0,0 +1,280 @@
+# Karpenter Blueprint: Prioritize Reserved Capacity
+
+## Purpose
+
+If you have purchased Savings Plans, Reserved Instances, or On-Demand Capacity Reservations (ODCRs), you want to prioritize this reserved capacity before using standard on-demand or spot instances. This blueprint demonstrates how to configure Karpenter to prioritize different types of reserved capacity, ensuring maximum utilization and cost optimization.
+
+This blueprint covers three main scenarios:
+1. **Savings Plans** - Prioritize instance families that match your Savings Plans
+2. **Reserved Instances** - Prioritize specific instance types with Reserved Instance commitments  
+3. **On-Demand Capacity Reservations (ODCRs)** - Utilize native ODCR support with the `reserved` capacity type
+
+## Requirements
+
+* A Kubernetes cluster with Karpenter installed. You can use the blueprint we've used to test this pattern at the `cluster` folder in the root of this repository.
+* For Savings Plans/Reserved Instances: A list of instance types or families that match your reservations, along with the total number of vCPUs reserved.
+* For ODCRs: Active On-Demand Capacity Reservations in your AWS account and Karpenter v1.3+ with the `ReservedCapacity` feature gate enabled.
+* A `default` Karpenter `NodePool` as that's the one we'll use in this blueprint. You did this already in the ["Deploy a Karpenter Default EC2NodeClass and NodePool"](../../README.md) section from this repository.
+
+## Scenario 1: Savings Plans Priority
+
+If you have Savings Plans for specific instance families (e.g., 20 vCPUs for `c5` family):
+
+**Scenario:** You purchased a Savings Plan covering 20 vCPUs of c5 instances. You want Karpenter to use those discounted c5 instances first, then fall back to regular pricing for any additional capacity needed.
+
+### Configuration
+
+```yaml
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: savings-plans
+spec:
+  limits:
+    cpu: "20" # Match your Savings Plan capacity
+  template:
+    metadata:
+      labels:
+        intent: apps
+    spec:
+      requirements:
+      - key: karpenter.k8s.aws/instance-family
+        operator: In
+        values:
+        - c5  # Instance family matching your Savings Plan
+      - key: karpenter.sh/capacity-type
+        operator: In
+        values:
+        - on-demand
+  weight: 100  # High priority
+```
+
+### Deploy
+
+```sh
+kubectl apply -f savings-plans.yaml
+kubectl apply -f workload-savings.yaml
+```
+
+### Results
+
+**Check the results:**
+```sh
+# Check nodes and NodePools
```

Comment:
do we need this line? I thought that the output was broken now I'm wonder if we really need it